DOCS.md

```diff
@@ -172,12 +172,13 @@ This posts weather information of a given location.
 
 __Usage__
 
-`@weather <zipcode | location name>`
+`@weather <zipcode | location name> > <degree type>`
 
 __Arguments__
 
 * `zipcode`: A five digit US zipcode.
 * `location name`: A letter and space only location name, or city name and state separated by a comma and a space.
+* `degree type`: A degree type to display the weather, the parameter can be 'C' for Celsius or 'F' for Fahrenheit. The default type is Fahrenheit.
```

Comment:
@rjchee The reason behind the flag is to sanitize the input. Right now we are polling the last character of the address, which is pretty sketch. Also, if we designate the flag syntax, then we can reuse old code after substringing the flag out, so we don't need to do deep regression testing.